### PR TITLE
[fdbshow]Adding more options for fdbshow and show mac

### PR DESCRIPTION
**Pull request by dgsudharsan**
Signed-off-by: Sudharsan Dhamal Gopalarathnam <sudharsand@nvidia.com> 

#### What I did
Adding more options for fdbshow and show mac

#### How I did it
Introducing additional options for address, type, count. Introduced addition argument in show mac for 

#### Previous command output (if the output of a command-line utility has changed)
```
show mac -h
Usage: show mac [OPTIONS]

  Show MAC (FDB) entries

Options:
  -v, --vlan TEXT
  -p, --port TEXT
  --verbose        Enable verbose output
  -h, -?, --help   Show this message and exit.
```
#### New command output (if the output of a command-line utility has changed)
```
show mac -h
Usage: show mac [OPTIONS] COMMAND [ARGS]...

  Show MAC (FDB) entries

Options:
  -v, --vlan TEXT
  -p, --port TEXT
  -a, --address TEXT
  -t, --type TEXT
  -c, --count
  --verbose           Enable verbose output
  -h, -?, --help      Show this message and exit.

Commands:
  aging-time

show mac
  No.    Vlan  MacAddress         Port         Type
-----  ------  -----------------  -----------  -------
    1      10  98:03:9B:82:BB:5B  Ethernet60   Dynamic
    2      10  EC:0D:9A:CD:91:72  Ethernet64   Dynamic
    3      10  EC:0D:9A:CD:91:73  Ethernet124  Dynamic
Total number of entries 3

show mac --address EC:0D:9A:CD:91:72
  No.    Vlan  MacAddress         Port        Type
-----  ------  -----------------  ----------  -------
    1      10  EC:0D:9A:CD:91:72  Ethernet64  Dynamic

show mac --count
Total number of entries 3

show mac --type Dynamic
  No.    Vlan  MacAddress         Port         Type
-----  ------  -----------------  -----------  -------
    1      10  98:03:9B:82:BB:5B  Ethernet60   Dynamic
    2      10  EC:0D:9A:CD:91:72  Ethernet64   Dynamic
    3      10  EC:0D:9A:CD:91:73  Ethernet124  Dynamic
Total number of entries 3

show mac aging-time
Aging time for switch is 600 seconds
```

### CODE REVIEW

File: show/main.py [hunk 1]
```diff
@@ -737,8 +744,29 @@ def mac(vlan, port, verbose):
...
+def aging_time(ctx):
+    app_db = SonicV2Connector()
+    app_db.connect(app_db.APPL_DB)
+    table = "SWITCH_TABLE*"
+    keys = app_db.keys(app_db.APPL_DB, table)
+
+    for key in keys:
+        fdb_aging_time = app_db.get(app_db.APPL_DB, key, 'fdb_aging_time')
+        if fdb_aging_time is not None:
+            click.echo("Aging time for {} is {} seconds".format(key.split(':')[-1], fdb_aging_time))
```
Comment (keboliu) [inline show/main.py:769]: I would suggest giving some explicit hints in case no ageing time is configured.

**Comment by keboliu** [comment]
another comment is that it's worth checking the community regression FDB test case: https://github.com/Azure/sonic-mgmt/blob/master/tests/fdb/test_fdb.py    to see whether it needs to be updated since the CLI output will be changed.

**Comment by dgsudharsan** [comment]
> another comment is that it's worth checking the community regression FDB test case: https://github.com/Azure/sonic-mgmt/blob/master/tests/fdb/test_fdb.py to see whether it needs to be updated since the CLI output will be changed.

Hi Kebo,
There is no change to existing CLI output.
The documentation was outdated and I added type. This PR just introduces additional filters and so existing tests should work as such.

File: show/main.py (hunk 1 above)
Comment (dgsudharsan) [reply show/main.py:773]: Done

**Review by keboliu — APPROVED** [review]

**Pull request closed without merging**